HISTORY.md

```diff
@@ -1,3 +1,16 @@
+4.1.0 / 2019-11-04
+
+  * deps: handlebars@4.5.1
+    - Add method Handlebars.parseWithoutProcessing
+    - Add guard to if & unless helpers
+    - Show source location for the strict lookup exceptions
+    - Bugfix: Use objects for hash value tracking
+    - Bugfix: Contents of raw-blocks must be matched with non-eager regex-matching
+    - Bugfix: prevent zero length tokens in raw-blocks
+    - Add missing type fields to AST typings
+    - Error message for syntax error missing location in 4.2.1+
```

Comment:
This module never let this bug through, as we worked around the issue until handbars fixed it. So probably not a change users would see fixed in the upgrade from hbs 4.0.6